apps/frontend/src/pages/landing/libs/components/popular-section/popular-section.tsx

```diff
@@ -5,12 +5,14 @@ import styles from "./styles.module.css";
 const PopularSection = (): React.JSX.Element => {
 	return (
 		<section className={styles["popular-section"]}>
-			<p className={styles["title"]}>Popular routes</p>
-			<ul className={styles["cards"]}>
-				<RouteCard imageUrl={null} name="Kralovska obora" />
-				<RouteCard imageUrl={null} name="Kralovska obora" />
-				<RouteCard imageUrl={null} name="Kralovska obora" />
-			</ul>
+			<div className={styles["popular-section-container"]}>
+				<p className={styles["title"]}>Popular routes</p>
```

Comment:
haven't noticed this previously, but this should be h2
```suggestion
				<h2 className={styles["title"]}>Popular routes</p>
```